### PR TITLE
Button

### PR DESCRIPTION
**Pull request by floraxue**
This PR finished the design system for all buttons. Buttons were added to the `_sass/components/_buttons.scss` file. 
This PR also added a testing page (accessible via http://127.0.0.1:4000/test/). See testing buttons on that page for the visual effects.

### CODE REVIEW

File: _sass/components/_buttons.scss [hunk 1]
```diff
@@ -22,4 +23,104 @@
     //box-shadow: 0 7px 14px rgba(50, 50, 93, 0.1), 0 3px 6px rgba(0, 0, 0, 0.08);
     text-decoration: none;
   }
+  &:active {
+    color: #ffffff;
```
Comment (tslmy) [inline _sass/components/_buttons.scss]:
```suggestion
    color: $white;
```

File: _sass/components/_buttons.scss [hunk 2]
```diff
@@ -22,4 +23,104 @@
...
+    background-color: lighten($primary, 50%);
+    transform: translateY(-1px);
+    //box-shadow: 0 7px 14px rgba(50, 50, 93, 0.1), 0 3px 6px rgba(0, 0, 0, 0.08);
```
Comment (tslmy) [inline _sass/components/_buttons.scss]:
```suggestion
```

File: _sass/components/_buttons.scss [hunk 3]
```diff
@@ -22,4 +23,104 @@
     //box-shadow: 0 7px 14px rgba(50, 50, 93, 0.1), 0 3px 6px rgba(0, 0, 0, 0.08);
```
Comment (tslmy) [inline _sass/components/_buttons.scss]:
```suggestion
```

**Review by tslmy — CHANGES_REQUESTED** [review]

File: _sass/components/_buttons.scss (hunk 3 above)
Comment (floraxue) [reply _sass/components/_buttons.scss]: I didn't want to change the legacy code that was copied from Serif (and it's gonna be deprecated anyways).

File: _sass/components/_buttons.scss (hunk 1 above)
Comment (floraxue) [reply _sass/components/_buttons.scss:27]: Ditto.

Comment (floraxue) [reply _sass/components/_buttons.scss]: Removed this whole block. Testing code not intended to be submitted.

File: _sass/components/_buttons.scss (hunk 2 above)
Comment (floraxue) [reply _sass/components/_buttons.scss]: Removed this whole block. Testing code not intended to be submitted.

**Review by tslmy — APPROVED** [review]

**Pull request merged**